NEWS.md

```diff
@@ -1,5 +1,65 @@
 # OSBuild - Build-Pipelines for Operating System Artifacts
 
+## CHANGES WITH 24:
+
+  * Add a new `org.osbuild.rhsm` stage to configure the Red Hat
+    subscription management. Currently the stages supports only
+    enabling or disabling the RHSM DNF plugins.
+
+  * A new `org.osbuild.sysconfig` stage was added that allows
```

Comment:
Hehe all the other news are written in imperative form. (Yes, feeling nitpicky tonight.)